test.py

```diff
@@ -0,0 +1,47 @@
+import os
+
+import requests
+
+example = """
+assistant: Hi Alex, nice to meet you. I've just had a look at your intake form. Can you tell me a bit more about yourself, and maybe why you're here?
+user: I'm thirty five years old and I I'm here because I am done with depression and don't have a job right now.
+"""
+
+output_dir = 'results'
+os.makedirs(output_dir, exist_ok=True)
+
+
+def write_to_file(filename, content):
+    with open(os.path.join(output_dir, filename), 'w') as file:
+        file.write(content)
+
+
+# Test the /anonymize endpoint
```

Comment:
It'd be better to convert this script to an actual set of unit tests so we can run them as part of the CI pipeline and have confidence that the service is working.

Writing proper tests is also great practice if you haven't done so before.

You can find guidance on testing in FastAPI [here](https://fastapi.tiangolo.com/tutorial/testing/) but it can be fiddly - if you get stuck we can pair on it just lmk!

Reply:
Added some simple tests in test_main.py. Since the outputs are not deterministic, I am just validating inputs and output request keys

Reply:
Nice 👍 

Generally for unit tests you'd want to mock things that are non-deterministic or depend on external services (a call to Open AI is a good examples of both) - but don't worry about this for now.